hs_access_control/models/resource.py

```diff
@@ -53,24 +53,67 @@ class ResourceAccess(models.Model):
     # workalike queries adapt to old access control system
     #############################################
 
+    @property
+    def __view_users_from_individual(self):
+        return Q(is_active=True,
+                 u2urp__resource=self.resource)
+
+    @property
+    def __view_users_from_group(self):
+        return Q(is_active=True,
+                 u2ugp__group__gaccess__active=True,
+                 u2ugp__group__g2grp__resource=self.resource)
+
+    @property
+    def __view_users_from_community(self):
+        return Q(is_active=True,
+                 u2ugp__group__gaccess__active=True,
```

Comment:
May be this filter condition is not needed: `u2ugp__group__gaccess__active=True`

Reply:
I don't know if the argument being passed has an active group or not.